javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,133 @@
+let myName = "";
+const toDoList = [];
+
+function getReply(command) 
+{
+    let x = command.toLowerCase();
```

Comment:
`x` isn't a descriptive variable name. 
If you want, you can reassign the `command` variable:
`command = command.toLowerCase()`

Reply:
yes i will try to use descriptive variable names in the future coding, thanks a lot